source/clear-linux/get-started/install-configfile.rst

```diff
@@ -0,0 +1,119 @@
+.. _install-configfile:
+
+Install using clr-installer and a configuration file
+####################################################
+
+This guide shows you how to install |CL-attr| using the clr-installer tool
+with a configuration file. This method allows you to predefine configurations
+and options, which are then fed into clr-installer, thus relieving you of
+having to walk through the installer step by step. You can save the
+configuration file and reuse it again.
+
+System requirements
+*******************
+
+Ensure that your target system supports the installation:
+
+* :ref:`system-requirements`
+* :ref:`compatibility-check`
+
+Process
+*******
+
+Perform the following steps:
+
+#. Download the latest :file:`live-server.iso` file.
+
+   For example:
+   https://download.clearlinux.org/releases/29690/clear/clear-29690-live-server.iso.xz
+
+#. Uncompress it and burn it to a USB thumb drive.
+#. Boot up the USB thumb drive.
+#. Select :guilabel:`Clear Linux OS` from the menu.
+#. After it boots, press :guilabel:`<CTRL><Shift><F2>` to bring up a TTY console.
+#. Log in as root and set a password.
+#. Verify you have an IP address (network connection) and configure proxies (if needed).
+#. Download an :file:`installer.yaml` template.
+
+   For example:
+
+   .. code-block:: bash
+
+      # curl -O https://download.clearlinux.org/releases/29690/clear/config/image/installer.yaml
+
+#. Edit the template and change the settings as needed. Commonly changed settings include:
+
+   * Under *block-devices*, set “file: "/dev/sda"” or enter your preferred device.
+   * Under *targetMedia*, set the third partition size to “0” to use the entire disk space.
+   * Under *bundles*, add additional bundles as needed.
+   * Delete the *kernel-arguments* section unless you want to add kernel parameters.
+   * Delete the *post-install* section unless you have post-installation scripts.
+   * Under *Version*, set a version number. Use “0” for the latest version.
+
+   Commonly changed settings are shown in lines 14, 20, 22, and 31 below.
+

```

Comment:
@bktan8  GitHub isn't rendering the sample config file properly, you have to build the docs locally to see the line numbers. Also, the highlighting may not display properly in the current docs template on the live site.